openshift/generate-manifests.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Know where the repo root is so we can reference things relative to it
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+# Source bingo so we can use kustomize and yq
+. "${REPO_ROOT}/.bingo/variables.env"
+
+# We're going to do file manipulation, so let's work in a temp dir
+TMP_ROOT="$(mktemp -p . -d 2>/dev/null || mktemp -d ./tmpdir.XXXXXXX)"
+# Make sure to delete the temp dir when we exit
+trap 'rm -rf $TMP_ROOT' EXIT
+
+# Copy all kustomize files into a temp dir
+TMP_CONFIG="${TMP_ROOT}/config"
+cp -a "${REPO_ROOT}/config" "$TMP_CONFIG"
+
+# Override namespace to openshift-operator-controller-system
+$YQ -i '.namespace = "openshift-operator-controller-system"' "${TMP_CONFIG}/default/kustomization.yaml"
```

Comment:
We named the catalogd namespace `openshift-catalogd`, so follow that pattern here?
```suggestion
$YQ -i '.namespace = "openshift-operator-controller"' "${TMP_CONFIG}/default/kustomization.yaml"
```